### ISSUE TITLE
Icons not showing on CloudBeaver interface while using Firefox

### ISSUE THREAD

**Issue by MaxVRAM** [dbeaver/cloudbeaver#478]
As the title suggestions. There are only a few icons that show in the GUI when I'm using CloudBeaver on Firefox. Version information and browser inspect log details below the example screenshots.

#### Firefox
![Firefox](https://user-images.githubusercontent.com/3183008/133707390-f94afcc7-ba1f-4669-8ffe-2095aaed5cf5.jpg)

#### Chrome
![Chrome](https://user-images.githubusercontent.com/3183008/133707397-1005ff76-7768-4127-9254-d1186bbf2565.jpg)

### System

**OS:** Ubuntu 20.04.3 LTS x86_64
**Kernel:** 5.11.0-34-generic
**Flavour:** Kubuntu
**DE:** Plasma / Breeze

**Browser 1:** Firefox 92.0 (64-bit)
**Browser 2:** Google Chrome Version 93.0.4577.82 (Official Build) (64-bit)

**Host:** Docker on Synology RS820+

### Logs / Details

While inspecting the page using Chrome, there's only a single warning about Roboto preloaded and not used.

In Firefox, however, there are a huge list of `Cookie "DBEAVER_SESSION_ID" will be soon treated as cross-side cookie...  ... because the scheme does not match.` warnings across many of the API and CSS files.

Also, in Firefox there is a single error message
```
TypeError: 'clipboard-read' (value of 'name' member of PermissionDescriptor) is not a valid value for enumeration PermissionName.
    updatePermissionStatus https://<my_url>/index.8c8eefd2d66b4a24bdde.js:10
    tryActivateClipboard https://<my_url>/index.8c8eefd2d66b4a24bdde.js:10
    load https://<my_url>/index.8c8eefd2d66b4a24bdde.js:10
    loadServices https://<my_url>/index.8c8eefd2d66b4a24bdde.js:10
index.8c8eefd2d66b4a24bdde.js:2:590399
    logException https://<my_url>/index.8c8eefd2d66b4a24bdde.js:2
    updatePermissionStatus https://<my_url>/index.8c8eefd2d66b4a24bdde.js:10
```

**Comment by kseniiaguzeeva** [comment]
Thank you for the report. But I cannot reproduce it. Could you please tell me CloudBeaver version?
![chrome_x7zDoKx2zT](https://user-images.githubusercontent.com/51405061/133889574-53ab81b0-e16a-451c-9d92-2f48203ab65b.png)
 We did some improvements for Firefox in 21.1.4 release.

**Comment by jathek** [comment]
I'm experiencing this as well. The icons load in Chrome (93.0.4577.82) and Edge (93.0.961.52) but not in Firefox (92.0).

I'm running the docker image with the following compose:
```yaml
  # CloudBeaver Community - Database manager with a rich web interface
  cloudbeaver:
    container_name: cloudbeaver
    image: dbeaver/cloudbeaver:21.1.4
    restart: unless-stopped
    networks:
      t2_proxy:
    # ports:
      # - 8978:8978
    environment:
      - TZ=$TZ
    volumes:
      - $DOCKERDIR/cloudbeaver:/opt/cloudbeaver/workspace
      - /mnt/user/downloads:/downloads:ro
    labels:
     ## Diun
      - "diun.enable=true"
      - "diun.watch_repo=true"
      - "diun.include_tags=^latest$$"
     ## Traefik
      - "traefik.enable=true"
      # HTTP Routers
      - "traefik.http.routers.cloudbeaver-rtr.entrypoints=websecure"
      - "traefik.http.routers.cloudbeaver-rtr.rule=Host(`cloudbeaver.$DOMAINNAME`)"
      - "traefik.http.routers.cloudbeaver-rtr-lan.entrypoints=websecure"
      - "traefik.http.routers.cloudbeaver-rtr-lan.tls.options=tls-opts@file"
      - "traefik.http.routers.cloudbeaver-rtr-lan.rule=Host(`cloudbeaver.$DOCKERHOSTNAME`)"
      # Middlewares
      - "traefik.http.routers.cloudbeaver-rtr.middlewares=cloudbeaver-headers@file,chain-authelia@file"
      - "traefik.http.routers.cloudbeaver-rtr-lan.middlewares=cloudbeaver-headers-lan@file,chain-authelia-lan@file"
      # HTTP Services
      - "traefik.http.routers.cloudbeaver-rtr.service=cloudbeaver-svc"
      - "traefik.http.routers.cloudbeaver-rtr-lan.service=cloudbeaver-svc"
      - "traefik.http.services.cloudbeaver-svc.loadbalancer.server.port=8978"
```
<table>
<tr>
	<td>
Product name: CloudBeaver CE Server</br>
Product description: Cloudbeaver Web UI Application</br>
Build time: September 9, 2021</br>
Backend version: 21.1.4.202109091449</br>
Frontend version: 21.1.4.202109091450</br>
	<td><img src="https://user-images.githubusercontent.com/692914/133898522-305190c3-319e-450b-a167-e816c5927445.png">
</table>

<table>
<tr>
	<td>Chrome Error Console
	<td>Firefox Error Console
<tr>
	<td><img src="https://user-images.githubusercontent.com/692914/133905095-9195f1cf-9ce9-4967-8827-5759a01aa736.png">
	<td><img src="https://user-images.githubusercontent.com/692914/133905243-2a95eaed-41a2-4402-af8f-ee8d86b0a237.png">

</table>

**Comment by kseniiaguzeeva** [comment]
Thank you. I reproduced it.

**Comment by MaxVRAM** [comment]
I feel like I provided everything except the critical CloudBeaver version number :laughing:  Here's the info from my CloudBeaver **panel.**

**Product name**
CloudBeaver CE Server
**Product description**
Cloudbeaver Web UI Application
**Build time**
September 9, 2021
**Backend version**
21.1.4.202109091449
**Frontend version**
21.1.4.202109091450****

Thanks for the quick response @kseniiaguzeeva

**Comment by kseniiaguzeeva** [comment]
The issue is fixed in the latest release 21.1.5 https://github.com/dbeaver/cloudbeaver.
